proto/frequenz/api/common/battery.proto

```diff
@@ -0,0 +1,216 @@
+// Contains definitions specific to batteries.
+//
+// Copyright:
+// Copyright 2023 Frequenz Energy-as-a-Service GmbH
+//
+// License:
+// MIT
+
+syntax = "proto3";
+
+package frequenz.api.common.battery;
+
+import "frequenz/api/common/common.proto";
+
+// Enumerated battery types.
+enum Type {
```

Comment:
We should just add component types here, as an extension to component categories.
So all this file should have is battery type enum, inverter type enum, ev charger type enum, etc.

Everything else like states, error codes, etc. are specific to the microgrid API.

Reply:
So maybe we could all all the types here, like `BatteryType`, `InverterType`, etc. And add these enums to `component_category.proto`, and rename it to `components.proto`.